ColorPicker/SwatchButton.js

```diff
@@ -112,7 +115,11 @@ const SwatchButtonBase = kind({
  * @public
  */
 const SwatchButtonDecorator = compose(
-	ButtonDecorator,
+	Pure,
+	UiButtonDecorator,
+	IconButtonDecorator,
+	TooltipDecorator({tooltipDestinationProp: 'decoration'}),
```

Comment:
I think we don't need to use these decorators for this component. Let's remove them and add `iconOnly` prop to `ButtonBase`.

Reply:
at line 101.